docs/apis/table.md

```diff
@@ -536,6 +536,29 @@ Table result = left.unionAll(right);
     </tr>
 
     <tr>
+      <td><strong>Minus</strong></td>
+      <td>
+        <p>Similar to a SQL EXCEPT clause. Returns elements from the first table that do not exist in the second table. Both tables must have identical schema, i.e., field names and types.</p>
```

Comment:
Please rephrase to "Except returns records from the first table that do not exist in the second table. Records that exist multiple times in the first table are returned exactly once, i.e., duplicates are removed.".